nupm/install.nu

```diff
@@ -23,107 +23,161 @@ def throw-error [
     }
 }
 
-def open-package-file [path: path] {
-    let package_file = $path | path join "package.nuon"
+def open-package-file [dir: path] {
+    let package_file = $dir | path join "package.nuon"
 
     if not ($package_file | path exists) {
-        throw-error $"package_file_not_found(ansi reset):\nno 'package.nuon' found in ($path)"
+        throw-error (
+            [
+                $"package_file_not_found(ansi reset):"
+                $"no 'package.nuon' found in ($dir)"
+            ]
+            | str join (char nl))
     }
 
     let package = open $package_file
 
     log debug "checking package file for missing required keys"
-    let required_keys = [$. $.name $.version $.description $.license $.type]
-    let missing_keys = $required_keys | where {|key| ($package | get -i $key) == null}
+    let required_keys = [$. $.name $.version $.type]
+    let missing_keys = $required_keys
+        | where {|key| ($package | get -i $key) == null}
```

Comment:
just curious about that one, you do not find
```nushell
let missing_keys = $required_keys | where {|key| ($package | get -i $key) == null}
```
easy enough to read?

Reply:
ooh, it's about the 80-char width you talk in the PR description :+1:

Reply:
Yeah, it's me being hardcore 80 chars.